rc/filetype/go.kak

```diff
@@ -87,8 +89,8 @@ define-command -hidden go-indent-on-new-line %~
         try %{ execute-keys -draft <semicolon><c-s>k<a-x> s ^\h*\K/{2,} <ret> y<c-o>P<esc> }
         # indent after a switch's case/default statements
         try %[ execute-keys -draft k<a-x> <a-k> ^\h*(case|default).*:$ <ret> j<a-gt> ]
-        # indent after if|else|while|for
-        try %[ execute-keys -draft <semicolon><a-F>)MB <a-k> \A(if|else|while|for)\h*\(.*\)\h*\n\h*\n?\z <ret> s \A|.\z <ret> 1<a-&>1<a-space><a-gt> ]
+        # indent after if|else|for
+        try %[ execute-keys -draft <semicolon><a-F>)MB <a-k> \A(if|else|for)\h*\(.*\)\h*\n\h*\n?\z <ret> s \A|.\z <ret> 1<a-&>1<a-space><a-gt> ]
```

Comment:
Hold on, this line should be dropped altogether.
Go does not have [if](https://golang.org/ref/spec#If_statements)/else/for statements without a `{`-prefixed block so they are already handled above.

(Also this indentation only triggers for parenthesized conditions like `if (x)` but in Go it's more idiomatic to write `if x`.)

Reply:
You are right, looks like someone copied it without understanding. I have removed this line.